app/helpers/host_description_helper.rb

```diff
@@ -1,5 +1,5 @@
 module HostDescriptionHelper
-  UI.register_host_description do
+  ::UI.register_host_description do
```

Comment:
@adamruzicka should we at least get these parts in and merged? They should be safe already. Eve if it doesn't solve it, it's good to get as much in as possible.

Reply:
Sure, I'll try salvaging what I can